tests: drivers: wifi: nrf_wifi: nrf70.build.offloaded_raw_tx fails on nrf7002dk/nrf5340/cpuapp

**Describe the bug**
The `tests/drivers/wifi/nrf_wifi` test suite, test `nrf70.build.offloaded_raw_tx` fails on `nrf7002dk/nrf5340/cpuapp` in CI. See https://github.com/zephyrproject-rtos/zephyr/actions/runs/13840443350/job/38726339049

Bisect points to a34198b2a243334f63cf8487bae19a7105f10fc1 as the commit introducing the error.

**To Reproduce**
Steps to reproduce the behavior:
1. `west build -p -b nrf7002dk/nrf5340/cpuapp tests/drivers/wifi/nrf_wifi -T nrf70.build.offloaded_raw_tx`
2. See error below.

**Expected behavior**
No CI failure.

**Impact**
CI failure.

**Logs and console output**
```
/__w/zephyr/modules/lib/nrf_wifi/fw_if/umac_if/src/offload_raw_tx/fmac_api.c
/__w/zephyr/modules/lib/nrf_wifi/fw_if/umac_if/src/offload_raw_tx/fmac_api.c: In function 'nrf_wifi_off_raw_tx_fmac_conf':
/__w/zephyr/modules/lib/nrf_wifi/fw_if/umac_if/src/offload_raw_tx/fmac_api.c:314:57: error: 'struct nrf_wifi_off_raw_tx_fmac_dev_ctx' has no member named 'country_code'
  314 |                                       dev_ctx_off_raw_tx->country_code);
      |                                                         ^~
ninja: build stopped: subcommand failed.
```

**Environment (please complete the following information):**

 - OS: Linux
 - Toolchain: Zephyr SDK
 - Commit SHA: 25f3de75a29a9267467de1df0baed5f6478332c8

**Additional context**
N/A

--- a/tests/drivers/wifi/nrf_wifi/testcase.yaml
+++ b/tests/drivers/wifi/nrf_wifi/testcase.yaml
@@ -4,6 +4,7 @@ common:
     - drivers
     - wifi
     - net
+    - nrf_wifi
   platform_allow:
     - nrf7002dk/nrf5340/cpuapp
 tests:

--- a/west.yml
+++ b/west.yml
@@ -321,7 +321,7 @@ manifest:
       revision: b84bd7314a239f818e78f6927f5673247816df53
       path: modules/bsim_hw_models/nrf_hw_models
     - name: nrf_wifi
-      revision: 0cd3bb2291a7dd22f4cdb1a4cd935a213c2bbfab
+      revision: 662ed74dce955e6c243f91c45a66768f5971e3cb
       path: modules/lib/nrf_wifi
     - name: open-amp
       revision: 52bb1783521c62c019451cee9b05b8eda9d7425f

